script/cibuild.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e # halt script on error
+
+bundle exec jekyll build
```

Comment:
It depends on what environment the deploy is targeted for. If non-production, as above; if production: `--config _config.yml,_config_production.yml`.